mws/apis/reports.py

```diff
@@ -2,12 +2,13 @@
 Amazon MWS Reports API
 """
 from __future__ import absolute_import
+from time import sleep
+from enum import Enum
```

Comment:
It seems with pip it got backported. That's why the tests work.
So long we keep it very basic it should work, i did test the basics:
<pre>python2.7
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type &quot;help&quot;, &quot;copyright&quot;, &quot;credits&quot; or &quot;license&quot; for more information.
&gt;&gt;&gt; from enum import Enum
&gt;&gt;&gt; class ReportType(Enum):
...     IN = &apos;something&apos;
... 
&gt;&gt;&gt; type(ReportType)
&lt;class &apos;enum.EnumMeta&apos;&gt;
&gt;&gt;&gt; ReportType.IN
&lt;ReportType.IN: &apos;something&apos;&gt;
&gt;&gt;&gt; ReportType.IN.value
&apos;something&apos;</pre>

Reply:
also you are right. It just appears that the import falls back to enum34 without doing anything.
Like one guy points out:
from enum import Enum     # for enum34, or the stdlib version
also in the [official docs for enum34](https://pypi.org/project/enum34/1.1.1/).
So we are lucky for once :)

Reply:
Our requirements file still lists `flake8`, and Travis CI is building requirements off that, so `enum34` becomes available as a result. Slight organizational problem there (whoops!), but not a big deal.

We'll still need a python<3.4 requirement for `enum34` in the setup in order to allow for pip deployment. Otherwise, it will break in production environments.